client/src/components/Sub_ProductDetail/StyleSelector.jsx

```diff
@@ -1,21 +1,24 @@
-import React, { useState } from 'react';
-import StyleThumbnails from './Sub_StyleSelector/StyleThumbnails.jsx';
+import React, { useState, useEffect } from 'react';
+import StyleList from './Sub_StyleSelector/StyleList.jsx';
 
-function StyleSelector({ currentStyle }) {
-  // console.log('SHOW CURRENT STYLE: ', currentStyle);
-  const [currentImageIndex, setCurrentImageIndex] = useState(0);
+function StyleSelector({ productID, productStyle, currentStyle, setCurrentStyle }) {
+  console.log('SHOW CURRENT STYLE: ', currentStyle);
+  const [startSelect, setStartSelect] = useState(false);
+
+  useEffect(() => { setStartSelect(false); }, [productID]);
 
   return (
-    <div className="style-slector">
-      <StyleThumbnails
-        images={currentStyle.photos}
-        currentImageIndex={currentImageIndex}
-        setCurrentImageIndex={setCurrentImageIndex}
-      />
-      <DefaultView
-        images={currentStyle.photos}
-        currentImageIndex={currentImageIndex}
-        setCurrentImageIndex={setCurrentImageIndex}
+    <div className="style-selector">
+      <div className="price" style={{ padding: '1rem 0' }}>
+        {currentStyle.sale_price ? <p><span className="sale-price">${currentStyle.sale_price}</span> <span className="original-sale-price">${currentStyle.original_price}</span></p> : <p><span className="original-price">${currentStyle.original_price}</span></p>}
```

Comment:
I think this ternary could be split into several lines to increase readability.

Reply:
Also deconstruction of the currentStyle object could work wonders here.

Reply:
good catch! i'll do it